support/peridio.json

```diff
@@ -0,0 +1,21 @@
+{
+    "version": 1,
+    "device_api": {
+      "certificate_path": "/etc/peridiod/peridio-cert.pem",
+      "url": "device.cremini.peridio.com",
+      "verify": true
+    },
+    "fwup": {
+      "devpath": "/dev/null",
+      "public_keys": []
+    },
+    "remote_shell": true,
+    "node": {
+      "key_pair_source": "env",
+      "key_pair_config": {
+        "private_key": "PERIDIO_PRIVATE_KEY",
+        "certificate": "PERIDIO_CERTIFICATE"
+      }
+    }
+  }
+  
```

Comment:
Newline at end unless intentional.

Reply:
intentional. New lines at the end of files is UNIX standard
https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline

Reply:
Yeah what I'm saying is this file is not following the standard.

<img width="87" alt="image" src="https://github.com/peridio/peridiod/assets/868014/4a8ceb3c-251e-48b1-880f-941a0aae9a3d">

In a pull request that red circle means the file does not currently end with a newline.

So I'm saying we should add one, unless not having one was intentional (which I doubt is the case).

Reply:
Weird, that wasn't showing in mine. I'll push again